class.jetpack-gutenberg.php

```diff
@@ -555,6 +555,11 @@ public static function enqueue_block_editor_assets() {
 			return;
 		}
 
+		// Required for Analytics. See _inc/lib/admin-pages/class.jetpack-admin-page.php.
+		if ( ! Jetpack::is_development_mode() && Jetpack::is_active() ) {
+			wp_enqueue_script( 'jp-tracks', '//stats.wp.com/w.js', array(), gmdate( 'YW' ), true );
+		}
```

Comment:
Without this, the tracks events would "fire" and be visible on the console, but nothing handled them.

This is duplicated from `jetpack-admin-page` as noted in the comment. Tracks events are now actually sent when they fire in the Gutenberg editor.